src/store/albums.js

```diff
@@ -165,38 +165,26 @@ export default {
   getters: {
     albums: (state) => Object.values(state.albums),
     albumsByTitle: (state, getters) =>
-      getters.albums.sort((a1, a2) =>
-        compareStrings(a1.normalized_title, a2.normalized_title)
-      ),
+      getters.albums.sort((a1, a2) => compareAlbumsByTitle(a1, a2)),
```

Comment:
```suggestion
      getters.albums.sort(compareAlbumsByTitle),
```
(Same for the others below.)